test/unit/spreadElement.spec.ts

```diff
@@ -64,3 +64,30 @@ test("Spread Element Iterable", () => {
         return JSONStringify(arr)`;
     expect(JSON.parse(util.transpileAndExecute(code))).toEqual([1, 2, 4, 8, 16, 32, 64, 128, 256]);
 });
+
+test.each(["", "string", "string with spaces", "string 1 2 3"])('Spread Element String "%s"', str => {
+    const code = `
+        const arr = [..."${str}"];
+        return JSONStringify(arr)`;
+    expect(JSON.parse(util.transpileAndExecute(code))).toEqual([...str]);
+});
+
+test.each([
+    "{ ...{ value: true } }",
+    "{ value: false, ...{ value: true } }",
+    "{ ...{ value: false }, value: true }",
+    "{ ...{ value: false }, value: false, ...{ value: true } }",
```

Comment:
Could you add a basic test where you destructure multiple different properties with different values and then expect JSON.parse toEqual?

Reply:
Also some where you have multiple destructures adding different properties to the same object.

Reply:
And I would also like to see a test where you destructure an identifier of a previously made object, rather than an object literal.